topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,424 @@
+---
+layout: tutorial_hands_on
+
+title: "metaQuantome 2: Function"
+zenodo_link: "https://doi.org/10.5281/zenodo.4110725"
+questions:
+ - "How do I look at differentially expressed proteins?"
+ - "How do I get top 5 or 10 function present in the sample?"
+ - "How can I visualize my results?"
+objectives:
+  - "A functional and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1h"
+key_points:
+  - "With function and Quantitation inputs perform statistical analysis"
+contributors:
+  - subinamehta
+  - timothygriffin
+  - pratikdjagtap
+  - emmaleith
+  - mariecrane
+  - pravs3683
+
+follow_up_training:
+
+    -
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+            - metaproteomics
+            
+        
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+            - metaquantome-taxonomy
+            
+requirements:
+
+   -
+   
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+           - metaquantome-data-creation
+            
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+metaQuantome {% cite Easterly2019 %} software suite was developed by the {% cite Galaxy-P %} for quantitative and statistical analysis of metaproteomics data. For taxonomic and functional expression analysis within the microbial community, metaQuantome leverages peptide-level quantitative information to generate visual outputs for data interpretation. It also generates outputs that help in understanding the taxonomic contribution to a selected function as well as functions expressed by selected taxonomic group. 
+
+In this tutorial, we will learn specifically about the metaQuantome Function workflow. In particular, we will learn about how peptide-level quantitation and associated functional information can be used to generate bar plots (for functional information such as molecular function, cellular components and biological processes), volcano plots (to detect differentially expressed function) and heatmap cluster analysis. To demonstrate the use of this workflow, we have used a thermophilic biogas reactor dataset wherein municipal food waste and manure is digested to generate methane gas (% cite Delogu2020 %)). After one round in the reactor, the microbial community was simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen three time points (8 hours, 18 hours and 33 hours) from this dataset.
```

Comment:
the citation (Delogu2020) uses parentheses instead of curly braces:

```suggestion
In this tutorial, we will learn specifically about the metaQuantome Function workflow. In particular, we will learn about how peptide-level quantitation and associated functional information can be used to generate bar plots (for functional information such as molecular function, cellular components and biological processes), volcano plots (to detect differentially expressed function) and heatmap cluster analysis. To demonstrate the use of this workflow, we have used a thermophilic biogas reactor dataset wherein municipal food waste and manure is digested to generate methane gas {% cite Delogu2020 %}). After one round in the reactor, the microbial community was simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen three time points (8 hours, 18 hours and 33 hours) from this dataset.
```